platform/mk2/capabilities.h

```diff
@@ -15,6 +15,7 @@
 #define USB_SERIAL_SUPPORT	1
 #define VIRTUAL_CHANNEL_SUPPORT	1
 #define WIFI_SUPPORT		1
+#define CAMERA_CONTROL      0
```

Comment:
MK2 gets no camera control?

Reply:
No. I anticipate in the future we may need custom firmware on the ESP8266, and MK2 has no way to flash custom firmware to wifi, since the boot and reset pins are not wired.